python/mxnet/libinfo.py

```diff
@@ -97,9 +97,16 @@ def find_include_path():
 
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     incl_path = os.path.join(curr_path, '../../include/')
```

Comment:
I think we should check pip path first. Most of the cases we should be able to find it there. And it might be the preferred path, as in Carl's horovod instruction, the include files are also copied under python/mxnet.

Reply:
Good suggestion. Updated